scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import { Task, TaskRunner } from './task';
+
+interface CloseMilestoneOptions {
+  milestone: string;
+}
+
+const closeMilestoneTaskRunner: TaskRunner<CloseMilestoneOptions> = async ({ milestone }) => {
+  const gitHubUsername = process.env.GITHUB_USERNAME;
+  const gitHubToken = process.env.GITHUB_ACCESS_TOKEN;
+
+  if (!gitHubUsername || !gitHubToken) {
+    console.log('the close milestone operation needs a GITHUB_USERNAME and  GITHUB_ACCESS_TOKEN environment variables');
```

Comment:
Can't think of a not-so-clever way to break this down but here is an option (feel free to disregard):

`console.log(
      'the close milestone operation needs a GITHUB_USERNAME and \
    GITHUB_ACCESS_TOKEN environment variables'.replace(/\s{2,}/g, ' ')
    );`

PS: there's two spaces before `GITHUB_ACCESS_TOKEN`

Reply:
Yeah... I'm happy with having it all in one line if the alternative is more code. I'll stick with my approach but thanks for the suggestion 🤝